### DISCUSSION TITLE
Current support for 2015 MBP

### DISCUSSION THREAD

**Discussion by Bachsau** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#694] [Q&A] [answered]
Hi, I still have an Early 2015 MacBook Pro which just got a new battery. Here on GitHub it is listed as a device with limited support, but on the website it is not listed at all. Does this mean support was dropped, or was it just ommited because of the limitations? And if so, could I still use "Sailing Mode" and the status icon if I bought it? Because I don't see a technical reason why this should not be possible on Intel based systems.

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @Bachsau,

Officially, the support for your MacBook model is currently limited to the Charge Limiter through Intel Mode. However, as described in this post #677 we are actually expanding support to your model soon and the current version of AlDente Pro works on your MacBook model even with all the other pro features such as the Sailing Mode. Official support will come soon.

Cheers,
Matthias

**Reply by Bachsau** [reply]
The latest version is awesome. I just bought a lifetime license. It looks like you removed the manual Intel mode switch in favor of using it as a fallback for when the OS is not in control. That's really great software design.
